backend/src/data_classes/Connection.py

```diff
@@ -3,18 +3,40 @@
 from datetime import datetime, timedelta
 from typing import List
 
+from src.config import DATETIME_FORMAT
 from src.data_classes.Transfer import Transfer
 from src.data_classes.Walk import Walk
 from src.data_classes.IAction import IAction
 
 
 @dataclass
 class Connection:
+    walking_only: bool
+    start_stop_name: str
+    end_stop_name: str
+    start_datetime: datetime
+    end_datetime: datetime
```

Comment:
these all should be methods or dynamic properties (`@property`), the current version doubles the information!